src/blocks/memory.rs

```diff
@@ -271,12 +313,50 @@ impl Memstate {
                 "SwapTotal:" => mem_state.swap_total = val,
                 "SwapFree:" => mem_state.swap_free = val,
                 "SwapCached:" => mem_state.swap_cached = val,
+                "Zswap:" => mem_state.zswap_compressed = val,
+                "Zswapped:" => mem_state.zswap_decompressed = val,
                 _ => (),
             }
 
             line.clear();
         }
 
+        // For ZRAM
+        for i in 0.. {
+            let zram_file_path = format!("/sys/block/zram{}/mm_stat", i);
+            match File::open(&zram_file_path)
+                .await
+                .error("/sys/block/zram[0-9]/mm_stat does not exist")
```

Comment:
This error is not used since it's just thrown away, but instead of using the match, we can keep the code stucture flatter by doing this:

```rust
            let Ok(file) = File::open(&zram_file_path).await else {
                break;
            };
```